cves/CVE-2013-4316.yml

```diff
@@ -162,37 +180,40 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
     note:
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: |
+      The default configuration values for the struts project were insecure
+      for the Dynamic Method Invokation feature. This should have defaulted
+      to false and had the user toggle it on if they needed it.
```

Comment:
In your last sentence, the use of the word "had" seems awkward. You could try something like, "this should have been defaulted to false, giving the user the option to toggle it on if needed." Consider using an active voice in this paragraph. You could also try being more general with your lesson.

Reply:
I've reworded this and also made the lesson a little more general.